<launch>
    <group ns="robot_nav">
	    <node pkg="reinovo_control" type="blank" name="blank" output="screen" required="true"/>
	</group>
	<!--以上勿动-->

	<!--move_base-->
	<include file="$(find oryxbot_navigation)/launch/move_base.launch"/>
	<!--amcl-->
	<include file="$(find oryxbot_navigation)/launch/amcl.launch"/>


</launch>
